docs/imagecustomizer/api/configuration/output.md

```diff
@@ -13,10 +13,28 @@ output:
   image:
     path: ./out/image.vhdx
     format: vhdx
+  verityHashes:
+    path: /output/verityhashes
+  ukis:
+    path: /output/ukis
+  bootloader:
+    path: /output/bootloader
```

Comment:
Are users likely to want to individually control these? An alternative would be to pass a single directory and have Prism create the three subdirectories under it